docs/basic-serialization.md

```diff
@@ -157,7 +155,7 @@ import kotlinx.serialization.json.*
 
 ### Backing fields are serialized
 
-Only class's properties with backing fields are serialized, so properties with getter/setter that don't
+Only a class's properties with backing fields are serialized, so properties with a getter/setter that don't
```

Comment:
I believe the article introduces the possessive noun in this case, which is singular. Turned around it would be _Only properties belonging to a class with backing fields_, not _Only properties belonging to class with backing fields_.